### DISCUSSION TITLE
i18n French translation for all files into locales folder.

### DISCUSSION THREAD

**Discussion by thomasbnt** [forem/forem#17107] [In Progress]
Hello ! 
I am in the process of translating the entire game into French. If you have other files and I didn't pay attention, let me know!
My actual fork : https://github.com/thomasbnt/forem/tree/i18n-french-locales-thomasbnt

![GitHub codespace, because Forem is really heavy to clone](https://user-images.githubusercontent.com/14293805/161612837-729b7255-3f28-485d-a6bf-b150c3791ee3.png)
_GitHub codespace, because Forem is really heavy to clone_

**Comment by thomasbnt** [comment]
PR as created 🎉

https://github.com/forem/forem/pull/17487

**Comment by paulVu** [comment]
thank for your translation, So if I want to translate to another language, Just do like you?

**Reply by thomasbnt** [reply]
Yes you can!
